<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>365</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 지출구분 항</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>44</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>65</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>63</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>64</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>신규등록 지출 항</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_costhang_widget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>63</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addcost_hang_button">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="costhang_save_button">
   <property name="geometry">
    <rect>
     <x>451</x>
     <y>210</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>312</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_costhang_tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>기존 등록된 지출 항</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="new_costhang_tableWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>161</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>신규 등록 지출 항</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="cost_delete_hang_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cost_delete_hang_button_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 수정</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addcost_hang_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costhang_reg_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costhang_save_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costhang_file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costhang_save_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cost_delete_hang_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>478</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cost_delete_hang_button_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costhang_modify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>costhang_reg_input()</slot>
  <slot>costhang_file_save()</slot>
  <slot>costhang_save_cancel()</slot>
  <slot>registed_hang()</slot>
  <slot>delete_selected_row()</slot>
  <slot>costhang_modify()</slot>
 </slots>
</ui>
